doc/run/k8s/install-sqlflow.yaml

```diff
@@ -64,3 +64,33 @@ spec:
         - /work/mysql-inited
       initialDelaySeconds: 5
       periodSeconds: 5
+  - name: model-zoo
+    image: sqlflow/sqlflow:model_zoo
+    imagePullPolicy: IfNotPresent
+    ports:
+    - containerPort: 50055
+      protocol: TCP
+    env:
+    - name: MYSQL_ADDR
+      value: 127.0.0.1:3306
+    volumeMounts:
+    # bind to host's docker.sock to build image
+    - mountPath: /var/run/docker.sock
```

Comment:
If the model zoo server runs in k8s, it will use kaniko to build image automatically.

Reply:
This config is for single-user playground only. Use local docker is to avoid introducing new image dependency and keep the vm small.

Reply:
I see.